index.js

```diff
@@ -348,6 +348,35 @@ bot.onText(/(spiderman|spider-man|spider man)/gi, function onEditableText(msg) {
+  const m = Math.floor(Math.random() * 6);
+  if (m <= 1) {
+    bot.sendMessage(msg.chat.id, msg);
+  }
+});
+
+bot.onText(/.*linux.*/gi, function onEditableText(msg) {
+  const msg = `I'd just like to interject for a moment. What you're referring to as Linux, is in fact, GNU/Linux, or as I've recently taken to calling it, GNU plus Linux. Linux is not an operating system unto itself, but rather another free component of a fully functioning GNU system made useful by the GNU corelibs, shell utilities and vital system components comprising a full OS as defined by POSIX.`
+  const msg1 = `Many computer users run a modified version of the GNU system every day, without realizing it. Through a peculiar turn of events, the version of GNU which is widely used today is often called "Linux", and many of its users are not aware that it is basically the GNU system, developed by the GNU Project.`
+  const msg2 = `There really is a Linux, and these people are using it, but it is just a part of the system they use. Linux is the kernel: the program in the system that allocates the machine's resources to the other programs that you run. The kernel is an essential part of an operating system, but useless by itself; it can only function in the context of a complete operating system. Linux is normally used in combination with the GNU operating system: the whole system is basically GNU with Linux added, or GNU/Linux. All the so-called "Linux" distributions are really distributions of GNU/Linux.`
+
+  const m = Math.floor(Math.random() * 10);
+  if (m <= 1) {
+    // Loops are forbidden here
+    bot.sendMessage(msg.chat.id, msg);
+    bot.sendMessage(msg.chat.id, msg1);
+    bot.sendMessage(msg.chat.id, msg2);
+  }
+});
+
```

Comment:
```suggestion
function genMessage(msgs, chanceFn) {
  return (msg) => {
    if (chanceFn()) msgs.forEach((m) => bot.sendMessage(msg.chat.id,m))
  }
}
bot.onText(/.*market.*/gi, genMessage(
  ['Easy, chief. Any rate the market offers is, by definition, fair.']),
  () => Math.floor(Math.random() * 3) <= 1
)

bot.onText(/.*gorilla.*/gi, genMessage(
  ["What the fuck did you just fucking say about me, you little bitch? I'll have you know I graduated top of my class in the Navy Seals, and I've been involved in numerous secret raids on Al-Quaeda, and I have over 300 confirmed kills. I am trained in gorilla warfare and I'm the top sniper in the entire US armed forces. You are nothing to me but just another target. I will wipe you the fuck out with precision the likes of which has never been seen before on this Earth, mark my fucking words. You think you can get away with saying that shit to me over the Internet? Think again, fucker. As we speak I am contacting my secret network of spies across the USA and your IP is being traced right now so you better prepare for the storm, maggot. The storm that wipes out the pathetic little thing you call your life. You're fucking dead, kid. I can be anywhere, anytime, and I can kill you in over seven hundred ways, and that's just with my bare hands. Not only am I extensively trained in unarmed combat, but I have access to the entire arsenal of the United States Marine Corps and I will use it to its full extent to wipe your miserable ass off the face of the continent, you little shit. If only you could have known what unholy retribution your little \"clever\" comment was about to bring down upon you, maybe you would have held your fucking tongue. But you couldn't, you didn't, and now you're paying the price, you goddamn idiot. I will shit fury all over you and you will drown in it. You're fucking dead, kiddo."],
  () => Math.floor(Math.random() * 6) <= 1
)
bot.onText(/.*linux.*/gi, genMessage(
  [`I'd just like to interject for a moment. What you're referring to as Linux, is in fact, GNU/Linux, or as I've recently taken to calling it, GNU plus Linux. Linux is not an operating system unto itself, but rather another free component of a fully functioning GNU system made useful by the GNU corelibs, shell utilities and vital system components comprising a full OS as defined by POSIX.`,
  `Many computer users run a modified version of the GNU system every day, without realizing it. Through a peculiar turn of events, the version of GNU which is widely used today is often called "Linux", and many of its users are not aware that it is basically the GNU system, developed by the GNU Project.`,
  `There really is a Linux, and these people are using it, but it is just a part of the system they use. Linux is the kernel: the program in the system that allocates the machine's resources to the other programs that you run. The kernel is an essential part of an operating system, but useless by itself; it can only function in the context of a complete operating system. Linux is normally used in combination with the GNU operating system: the whole system is basically GNU with Linux added, or GNU/Linux. All the so-called "Linux" distributions are really distributions of GNU/Linux.`],
    () => Math.floor(Math.random() * 10) <= 1
  )

```

Reply:
Good use of closures but my runtime doesn't support it

I run arch btw